src/Pester.RSpec.ps1

```diff
@@ -399,6 +399,9 @@ function New-PesterConfiguration {
       StackTraceVerbosity: The verbosity of stacktrace output, options are None, FirstLine, Filtered and Full.
       Default value: 'Filtered'
 
+      CIFormat: The CI format of output in build logs, options are None, Auto, AzureDevops and GithubActions. Environment variable TF_BUILD(https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml#system-variables-devops-services) is automically set to true when Azure Devops is detected. Environment variable GITHUB_ACTIONS(https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables) is automatically set to true when Github Actions is detected.
```

Comment:
I've added documentation here explaining what those environment vars do and why they are set to `True`. Let me know if I should be more explicit.

Reply:
I'd keep this short and simple + accepted values. How we auto-detect and how it looks should be added to https://pester.dev. For example in https://github.com/pester/docs/pull/128